src/import.ml

```diff
@@ -25,7 +25,7 @@ module Option = struct
 end
 
 module Or_error = struct
-  type 'a t = ('a, string) result
+  type 'a t = ('a, string) Result.t
```

Comment:
I think it'd be better to just add the alias to this import module ourselves:
```ocaml
type ('a, 'b) result = ('a, 'b) Result.t
```

Reply:
Changed